gradle.properties

```diff
@@ -1,7 +1,7 @@
-#Mon Jan 20 23:45:49 UTC 2020
-fiatVersion=1.12.0
+#Tue Dec 10 20:06:19 UTC 2019
+fiatVersion=1.13.0
 enablePublishing=false
+korkVersion=7.9.0
```

Comment:
Spinnaker bot is not working right now for the autobump, but you can bump this up to 7.13.1 here and this PR should pass.

Reply:
:+1: 
Waiting for Fiat to have its new version released.

Reply:
I just hit the release on fiat for 1.13.0